### PR TITLE
events: add account_id and principal information

### PR DESCRIPTION
**Pull request by warrn**
Adding new documented fields for account_id and principal in event envelopes.

Screenshots:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,6 @@ yarn.lock*
 package-lock.json
 
 *.zip
+
+# ide specific
+.idea
```
Comment (collinvandyck) [inline .gitignore:35]: nit: i put stuff like this in my gitignore global file

File: docs/events/event-sources.mdx [hunk 2]
```diff
@@ -15,12 +15,26 @@ Some event types support filters and selectable fields. Not all selectable field
...
+| id         | unique identifier for the principal, either a user or bot                                                  | `usr_2OtNv9qH5Nk4NuNeszZ39gBxZ4H`                                                                                     |
+| subject    | human readable unique identifier for the principal, either a user email or a bot name                      | `foo@example.com`                                                                                                     |
+| source     | mode of ingress into ngrok, typically Dashboard or API                                                     | `API`                                                                                                                 |
+| credential | id and uri resource for the credential used for authentication, only defined for programmatic access (API) | ```{"id": "ak_2Oxt94wYsBTLwFUoMZcJRvJTaub","uri": "https://api.ngrok.com/api_keys/ak_2Oxt94wYsBTLwFUoMZcJRvJTaub"}``` |
```
Comment (collinvandyck) [inline docs/events/event-sources.mdx:37]: API,  but also tunnels

**Review by collinvandyck — APPROVED** [review]
Nice. Smol suggestion about credential description, otherwise lgtm.

**Review by russorat — APPROVED** [review]
lgtm

File: docs/events/event-sources.mdx (hunk 2 above)
Comment (warrn) [reply docs/events/event-sources.mdx]: Done :)

File: .gitignore (hunk 1 above)
Comment (warrn) [reply .gitignore:35]: I should also do that, but also it's nice for others committing to not have to worry about it :)

**Pull request merged**